setup.py

```diff
@@ -18,7 +18,7 @@
 is_py3k = sys.version_info[0] == 3
 
 # -*- Distribution Meta -*-
-NAME = 'django-celery'
+NAME = 'django-celery-hp'
```

Comment:
ho means? reason for this change?

Reply:
Sorry I make a mistake in setup file. I will recommit and pull these again.